packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts

```diff
@@ -175,6 +175,7 @@ export class RichTextV2Control<
           this.editor.selection = null
           this.editor.children = this.defaultValue
           this.editor.onChange()
+          this.onLocalUserChange()
```

Comment:
This onChange in 177 doesn't have operations so it normally wouldn't be committed back to the builder. This overrides that for this situation. 

This message is received after the user clicks reset in the `Content` panel label.